core/market/decentralized/src/rest_api/provider.rs

```diff
@@ -43,7 +43,10 @@ async fn subscribe(
 
 #[actix_web::get("/offers")]
 async fn get_offers(market: Data<Arc<MarketService>>, id: Identity) -> HttpResponse {
-    HttpResponse::NotImplemented().finish()
+    match market.get_offers().await {
```

Comment:
You need to pass `id` down the road to filter out only offers subscribed by this Provider Agent, as @nieznanysprawiciel sharply noted